extensions/amp-position-observer/0.1/amp-position-observer.js

```diff
@@ -172,13 +173,15 @@ export class AmpVisibilityObserver extends AMP.BaseElement {
       'bottom-ratio': this.bottomRatio_,
       'top-margin': this.resolvedTopMargin_,
       'bottom-margin': this.resolvedBottomMargin_,
+      'has-exited': hasExited,
     };
     this.action_.trigger(this.element, name, event, ActionTrust.LOW);
-    if (this.useAnimationWorklet_ &&
-        !this.action_.hasAction(this.element, 'enter') &&
-        !this.action_.hasAction(this.element, 'exit')) {
-      this.maybeUninstallPositionObserver_();
-    }
+    //TODO(nainar): We want to remove the position observer if the scroll
```

Comment:
we should be able to remove `useAnimationWorklet_` now.

Reply:
The previous suggestion doesn't fix the problem of us doing the calculation twice (one in `amp-position-observer` and then in the `AnimationWorkletRunner`). 

What I have done is retained the calls to `triggerScroll_` as per usual. These are consumed by other components like `amp-3d-gltf` and `amp-animation` (non AnimationWorklet mode). The AnimationWorklet code will ignore the `triggerScroll_` since it will have already created the `AnimationWorkletRunner`. The percent values reported by `triggerScroll_` will be ignored since the `seekTo` action on `AnimationWorkletRunner` doesn't do anything. 

Does this make sense?